.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -204,6 +211,21 @@ if [ "${WORKER}" == 0 ]; then
     chmod  770                                      $SUBMITTY_DATA_DIR/to_be_built
 fi
 
+#Add the submitty user to /etc/sudoers if in worker mode.
+if [ "${WORKER}" == 1 ]; then
+    if ! grep -q "${SUBMITTY_SUPERVISOR}" /etc/sudoers; then
+        echo "" >> /etc/sudoers
+        echo "#grant the submitty user on this worker machine access to install submitty" >> /etc/sudoers
+        echo "%${SUBMITTY_SUPERVISOR} ALL = (root) NOPASSWD: ${SUBMITTY_INSTALL_DIR}/.setup/INSTALL_SUBMITTY.sh" >> /etc/sudoers
+        echo "#grant the submitty user on this worker machine access to the systemctl wrapper" >> /etc/sudoers
+        echo "%${SUBMITTY_SUPERVISOR} ALL = (root) NOPASSWD: ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils/systemctl_wrapper.py" >> /etc/sudoers
+    fi
+fi
+
+echo "#grant the hwcron user access to the daemon utilities" >> /etc/sudoers
+echo "%${HWCRON_USER} ALL = (root) NOPASSWD: ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils/systemctl_wrapper.py" >> /etc/sudoers
```

Comment:
This probably should be in `install_system.sh` as right now, this will just append these lines each time you run `INSTALL_SUBMITTY.sh` which we probably don't want for `/etc/sudoers`.

Reply:
Talked about this in person, moving the lines to `install_system.sh` and wrapping a missing check around it.